dkube/patches/cache-server-mysql-secret.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cache-server
+  namespace: kubeflow
+spec:
+  template:
+    spec:
+      containers:
+        - name: server
+          env:
+            - name: DBCONFIG_USER
+              valueFrom:
+                secretKeyRef:
+                  name: katib-mysql-secrets
+                  key: DBCONFIG_USER
+            - name: DBCONFIG_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mysql-db-pxc-db
+                  key: root
```

Comment:
why is key 'root' ? can it be DBCONFIG_PASSWORD ?

Reply:
@AnsuFrancis

Reply:
The key in the mysql-db-pxc-db secret is root. Like
```
Data:
    root: cGFzc3dvcmQ=
```
Since it is created by pxc mysql, we can't change that.